nginx.conf

```diff
@@ -46,6 +47,13 @@ http {
     }
 
     location / {
+      # more extensions can be added inside the parentheses, separated by | characters
```

Comment:
Do you mind adding some more info in a comment here about what this does and why we added it?

Reply:
https://github.com/18F/federalist-proxy/pull/23

Reply:
beautiful